src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,50 @@
+.nav {
+  flex-shrink: 0;
+
+  &__list {
+    list-style: none;
+
+    margin: 0;
+    padding: 0;
+    display: flex;
+  }
+
+  &__item {
+    position: relative;
+
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+  }
+
+  &__link {
+    color: $main-text-color;
```

Comment:
Ensure that the nav links have the correct color `#060b35` as required by the task. The `$main-text-color` variable is correctly set to `#060b35`, so this should be fine.